.github/actions/workflow-vars/action.yml

```diff
@@ -1,17 +1,25 @@
 name: Workflow vars
 description: Set common vars for workflow
 inputs:
-  persistent:
-    description: Don't use changing vars like SHA commit
-    # TODO: Change that to real bool when it's supported in GHA.
-    default: "false"
   setup-flow:
     description: The chart setup flow either "install" or "upgrade".
     default: "install"
   ingress-hostname-base:
     description: The base of the Ingress hostname.
   platform:
     description: The deployment cloud platform like GKE or ROSA.
+  ttl:
+    description: |
+      Define a ttl for the lifespan of the workflow
```

Comment:
We need to make this clear, it's not the workflow lifespan, it's the namespace/deployment lifespan.
Let's call it `deployment-ttl` and adjust the description accordingly.

Reply:
✅